app/helpers/application_helper.rb

```diff
@@ -26,4 +26,38 @@ def title(page_title = nil)
   def visually_hidden_service(service_name)
     tag.span service_name.humanize(capitalize: false), class: 'visually-hidden'
   end
+
+  def stanford_only_icon
+    tag.button(class: 'stanford-only btn',
+               data: { 'bs-toggle': 'popover', 'bs-placement': 'right',
+                       'bs-content': 'Available to Stanford-affiliated users only. Log in to access.' },
+               aria: { label: 'Stanford-only' }) do
+      tag.span(aria: { label: 'Stanford-only' }) do
+        tag.svg(width: "14px", height: "14px", viewBox: '0 0 14 14', aria: { hidden: true }) do
```

Comment:
Do we need to deconstruct the SVG with rails helpers here? Can we just leave it in a string?